Esignet/README.md

```diff
@@ -137,15 +153,34 @@
 
 *  OIDC - Configuration (Execution) : Open ID Connect dynamic provider discovery is not supported currently, this endpoint is only for facilitating the OIDC provider details in a standard way.
 
-*  OIDC - JSON Web Key Set (Execution) : Endpoint to fetch all the public keys of the eSignet server.Returns public key set in the JWKS format.
+*  OIDC - JSON Web Key Set (Execution) : Endpoint to fetch all the public keys of the Esignet server.Returns public key set in the JWKS format.
 
 
 ### Execution points for eSignet Wallet Binding API's
 
+
 *  Wallet Binding - Send Binding OTP (Execution) : This thread group will send the otp for the individual id passed in the request body.  For Registered individual id we have separately added the setup thread group for creating identities.
 
 *  Wallet Binding : 
    * Wallet Binding (Preparation) - This preparation thread group contains binding otp api from which we will save the passed individual id into a file and will use that same file in the execution.
 
-   * Wallet Binding (Execution) - In this thread will pass the auth factor type as "WLA". Also, a JWT format binding public key which will be generated from a code written in JSR223 preprocessor. Will use the file generated from the preparation and it can't be used multiple times.
+   * Wallet Binding (Execution) - In this thread will pass the auth factor type as "WLA". Also, a JWT format binding public key which will be generated from a code written in JSR223 preprocessor. Will use the file generated from the preparation and it can't be used multiple times. 
+
+### Execution points for eSignet Sign Up Services API's
+
+* Sign Up Service - Setting (Execution) : This thread only contains a Setting endpoint API which is a GET APi. 
+
+* Sign Up Service - Generate Challenge (Execution) : This thread contains Generate Challenge endpoint API. We need to pass an identifier value which is nothing but a 8-10 digit phone number with country code as the prefix. We are using a preprocessor from which we are getting the random generated phone number.
+
+* Sign Up Service - Verify Challenge (Praparation) : In this thread we have generate challenge API from which we will get the value of identifier and from response headers will get the transaction id which wil be stored in a csv file.
+
+* Sign Up Service - Verify Challenge (Execution) : This thread contains verify challenge API in which we will pass the value of identifier i.e. phone number and transaction id in the headers which will get from the csv file generated in preparation. The file generated can be used for multiple iterations. 
```

Comment:
The file generated can be used for multiple iterations. - I believe this is not true

Reply:
Updated.